test/metabase/test/data/interface.clj

```diff
@@ -290,10 +290,12 @@
   "Return the connection details map that should be used to connect to the Database we will create for
   `database-definition`.
 
+  `connection-type` is either:
+
   *  `:server` - Return details for making the connection in a way that isn't DB-specific (e.g., for
                  creating/destroying databases)
   *  `:db`     - Return details for connecting specifically to the DB."
-  {:arglists '([driver context database-definition])}
+  {:arglists '([driver connection-type database-definition])}
```

Comment:
Unrelated to the core changes here, but I changed `context` to `connection-type` to make it a little more obvious what the purpose of this parameter was. `context` could really mean almost anything (is it a stateful map of some sort? etc.) whereas `connection-type` is a little less ambiguous.